server/api/routes/bookmark.js

```diff
@@ -0,0 +1,14 @@
+import { Router } from 'express';
+import passport from 'passport';
+// import {
+//   newArticleValidator, createHighlightValidator, ratingValidator
+// } from '../middlewares/validation/article';
+import { createBookmark, deleteBookmark, getUserBookmarks } from '../controllers/bookmark';
+
+const bookmarksRouter = Router();
+
+bookmarksRouter.post('/articles/:articleId/bookmark', passport.authenticate('jwt', { session: false }), createBookmark);
```

Comment:
Hi @oyedejipeace. Good Job with your task. However, I see you named the bookmark endpoint here with `/bookmark`. Following REST standard which we have adopted, I believe these should be `bookmarks` ie *plural* instead of *singular*.

Endeavour to effect the changes in your test scripts as well.

Reply:
Sam thank you very much for your observation. The standard followed is REST standard. Kindly checkout the link below for more info on the REST standard.
https://restfulapi.net/resource-naming/

Reply:
Ok. I understand. `bookmark` is not a resource but an action (function) and the request returns the bookmarked article. Sorry, I didn't get that initially.